### PR TITLE
Add new personal bio

### PR DESCRIPTION
**Pull request by EricKolibacz**

### CODE REVIEW

File: mentors/python/e.json [hunk 1]
```diff
@@ -1 +1,8 @@
-[]
+[{
```
Comment (rpottsoh) [inline mentors/python/e.json]:
```suggestion
[
  {
```
Small formatting changes.

File: mentors/python/e.json [hunk 2]
```diff
@@ -1 +1,8 @@
...
+  "github_username": "EricKolibacz",
+  "name": "Eric Kolibacz",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "Python was the first programming language I got engaged with, back in the days when I was still in high school. Exersicm's mentoring program is a great oppurtunity to give somehting back to community while improving my own coding skill."
```
Comment (rpottsoh) [inline mentors/python/e.json]: Indent these lines so that they each 2 spaces in from the `{`.

File: mentors/python/e.json [hunk 3]
```diff
@@ -1 +1,8 @@
...
+}]
```
Comment (rpottsoh) [inline mentors/python/e.json]:
```suggestion
  }
]
```
Should mirror the top of the file essentially.

File: mentors/python/e.json (hunk 1 above)
Comment (EricKolibacz) [reply mentors/python/e.json]: Thanks for the help :)

**Comment by EricKolibacz** [comment]
Great thanks for the help :)

**Comment by rpottsoh** [comment]
@EricKolibacz clicking the `Resolve Conversation` button doesn't resolve anything, it is just a marker.  You still need to resolve the issues.  Once you push another commit GutHub will mark the conversations reolved for you.

**Comment by EricKolibacz** [comment]
Sorry for being so clumsy, but I am not 100% sure how to properly push a commit to GitHub. Do I need to edit the file on the GitHub website or pull the file on my local computer edit, add and commit it and then push it?

**Comment by rpottsoh** [comment]
Since you have created another PR I am going to close this one.

**Pull request closed without merging**